### PR TITLE
set stop_gradient sign to all loss layers

### PR DESCRIPTION
**Pull request by velconia**
test=develop

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -1472,6 +1472,7 @@ def bpr_loss(input, label, name=None):
 
     helper = LayerHelper('bpr_loss', **locals())
     out = helper.create_variable_for_type_inference(dtype=input.dtype)
+    label.stop_gradient = True
```
Comment (chengduoZH) [inline python/paddle/fluid/layers/nn.py:1475]: **We shouldn't secretly change the user's configuration.**

Comment (velconia) [reply python/paddle/fluid/layers/nn.py:1475]: This is a hot fix

Comment (chengduoZH) [reply python/paddle/fluid/layers/nn.py:1475]: **This is not hot fix. It's introducing new bugs.**

Comment (velconia) [reply python/paddle/fluid/layers/nn.py:1475]: So... what's the best way to solve the problem?

**Pull request closed without merging**